packages/nextjs/app/debug/_components/contract/TupleArray.tsx

```diff
@@ -97,12 +97,12 @@ export const TupleArray = ({ abiTupleParameter, setParentForm, parentStateObject
 
   return (
     <div>
-      <div className="collapse collapse-arrow">
+      <div className="collapse collapse-arrow bg-base-200 pl-2 py-1.5 mt-1 border-2 border-secondary">
```

Comment:
Should we delete `mt-1` to have the same spacing between type text and border as regular inputs?